airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/steps/docker.py

```diff
@@ -55,13 +55,21 @@ def _mount_paths(self, container: dagger.Container) -> dagger.Container:
             if path_to_mount.optional and not path_to_mount.get_path().exists():
                 continue
 
-            path_string = str(path_to_mount)
-            destination_path = f"/{path_string}"
-            if path_to_mount.is_file:
-                file_to_load = self.context.get_repo_file(path_string)
-                container = container.with_mounted_file(destination_path, file_to_load)
-            else:
-                container = container.with_mounted_directory(destination_path, self.context.get_repo_dir(path_string))
+            if path_to_mount.get_path().is_symlink():
+                container = self._mount_path(container, path_to_mount.get_path().readlink())
+
```

Comment:
isn't an `else` missing there?

Reply:
no, if it's not a symlink, then line 61 is enough

Reply:
L61 mounts the symlink itself. It's one of the quirks of buildkit that it doesn't resolve the symlink before mounting it.

Reply:
we're not the only people to be annoyed by this: https://github.com/moby/moby/issues/1676